DuckDuckGo/NetworkProtectionStatusView.swift

```diff
@@ -151,8 +176,8 @@ struct NetworkProtectionStatusView: View {
 
     @ViewBuilder
     private func locationDetails() -> some View {
-        if !statusModel.isSnoozing, let location = statusModel.location {
-            Section {
+        Section {
```

Comment:
Changed the code a bit to move the section out of the `if` conditional block, as it was shared in both scenarios.

This cleans up the code a bit and allows me to add the same `TipView` below for both branches of the `if` condition.